app/tasks/email.py

```diff
@@ -72,16 +68,16 @@ def send_email(to_email_addresses, email_body, subject, from_email, from_name, r
         return_address = current_app.config.get('DM_EMAIL_RETURN_ADDRESS')
 
         result = email_client.send_email(
-            Source=u"{} <{}>".format(from_name, from_email),
+            Source="{} <{}>".format(from_name, from_email),
             Destination=destination_addresses,
             Message={
                 'Subject': {
-                    'Data': subject,
+                    'Data': subject.decode(),
```

Comment:
should use subject straightaway without calling to_bytes() and decode()

Reply:
I think it's ok to leave this as is.  Emails are being sent and received.  We could change this but we'd have to test again.